extensions/wikia/SiteWideMessages/templates/editor.tmpl.php

```diff
@@ -163,6 +163,31 @@
 								<?= wfMsg('swm-label-mode-users-group-hint') ?>
 							</td>
 						</tr>
+						<tr>
+							<td>
+								<input name="mSendModeUsers" id="mSendModeUsersPU" type="radio" value="POWERUSER"<?= $formData['sendModeUsers'] == 'POWERUSER' ? ' checked="checked"' : ''?>/>
+							</td>
+							<td>
+								<label for="mSendModeUsersPU"><?= wfMessage( 'swm-label-mode-users-poweruser' )->escaped() ?></label>
+							</td>
+							<td>
+								<?php
+									foreach	( $formData['powerUserTypes'] as $powerUserType ) {
```

Comment:
Where is `powerUserTypes` set on the form data? I don't see these checkboxes being added only the single send mode option.